tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -787,59 +696,25 @@ module.exports = {
     filesListNoContentMessage: {
       selector: '#files-list-container .files-list-no-content-message'
     },
-    fileActionsButtonInFileRow: {
-      selector: '//button[contains(@class, "files-list-row-show-actions")]',
-      locateStrategy: 'xpath'
-    },
     deleteFileConfirmationDialog: {
       selector: '#delete-file-confirmation-dialog',
       locateStrategy: 'css selector'
     },
-    deleteButtonInFileRow: {
-      selector: '//button[@aria-label="Delete"]',
-      locateStrategy: 'xpath'
-    },
-    downloadButtonInFileRow: {
-      selector: '//button[@aria-label="Download"]',
-      locateStrategy: 'xpath'
-    },
-    restoreButtonInFileRow: {
-      selector: '//button[@aria-label="Restore"]',
-      locateStrategy: 'xpath'
-    },
-    deleteImmediatelyButtonInFileRow: {
-      selector: '//button[@aria-label="Delete"]',
-      locateStrategy: 'xpath'
-    },
     deleteFileConfirmationBtn: {
       selector: '#oc-dialog-delete-confirm'
     },
     shareButtonInFileRow: {
       selector: '//button[@aria-label="Collaborators"]',
       locateStrategy: 'xpath'
     },
-    renameFileConfirmationDialog: {
-      selector: '#change-file-dialog'
-    },
-    renameButtonInFileRow: {
-      selector: '//button[@aria-label="Rename"]',
-      locateStrategy: 'xpath'
-    },
-    renameFileInputField: {
-      selector: '//div[@id="change-file-dialog"]//input',
-      locateStrategy: 'xpath'
-    },
-    renameFileConfirmationBtn: {
-      selector: '#oc-dialog-rename-confirm'
-    },
     fileRowByName: {
       selector: '//span[@class="oc-file-name"][text()=%s and not(../span[@class="oc-file-extension"])]/../../../../../div[@data-is-visible="true"]'
     },
     fileRowByNameAndExtension: {
       selector: '//span[span/text()=%s and span/text()="%s"]/../../../../div[@data-is-visible="true"]'
     },
     fileLinkInFileRow: {
-      selector: '//span[contains(@class, "file-row-name")]'
+      selector: '//span[@role="button"]'
```

Comment:
why do we have to change that?

Reply:
just to see if issue #36089 is affetcted!
reverted back to original...